aws/src/services/lambdaCloudService.ts

```diff
@@ -80,11 +79,11 @@ export class LambdaCloudService implements CloudService {
 
   /**
    *
-   * @param name Name of Lambda function to invoke
-   * @param fireAndForget Wait for response if false (default behavior)
-   * @param payload Body of HTTP request
+   * @param params invoke interface with parameters needed
    */
-  public async invoke<T>(name: string, fireAndForget, payload: any): Promise<T> {
+  public async invoke<T>(params: InvokeRequest): Promise<T> {
+    const { name, fireAndForget, payload} = params;
```

Comment:
nit: add space between `payload` and the `curly brace`

Reply:
Added this space, thanks for this feedback